weave/trace_server/trace_server_interface.py

```diff
@@ -191,6 +191,10 @@ class CallsQueryReq(BaseModel):
     sort_by: typing.Optional[typing.List[_SortBy]] = None
     query: typing.Optional[Query] = None
 
+    # TODO: type this with call schema columns
+    # optional column_specification, selection and order
```

Comment:
note: assuming `column_specification` refers to some future state where we support things like adding/subtracting/etc... then the typing would be `list[str | SOMETHING]`. Totally fine, just want to make sure i understand what that means.